packages/tooltip/stories/Tooltip.stories.tsx

```diff
@@ -107,19 +109,14 @@ export const MinOrMaxWidth = () => (
   </>
 );
 
-export const ContentWidth = () => (
+export const ContentWidth = args => (
```

Comment:
I'm getting a console error here as well when trying to change the `preferredDirection` option on this story. The rest of the stories in this component seem to be fine though:

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'y')`